exercises/practice/accumulate/description.md

```diff
@@ -0,0 +1,26 @@
+# Description
```

Comment:
A practice exercise does not have a `description.md` file. See [the docs](https://exercism.org/docs/building/tracks/practice-exercises). Could you remove this file?